src/Components/getList.js

```diff
@@ -27,16 +27,67 @@ function EmptyList() {
   );
 }
 
-function FullList(props) {
+
+const FullList = (props)=> {
   const [isChecked, setCheck] = useState(false);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleChange = e => {
+    setSearchTerm(e.target.value);
+  };
+
+  const clearSearch = () => {
+    setSearchTerm("");
+  };
+
+  const searchList = props.data.filter(item => {
+    return item.itemName.toLowerCase().includes(searchTerm.toLowerCase());
```

Comment:
It's great that this feature works without worrying about matching case when searching! Nice work 🎉